### ISSUE TITLE
How can i use the SpanBert to predict a span of tokens in some specific contex?

### ISSUE THREAD

**Issue by hzy312** [facebookresearch/SpanBERT#78]
For example, the sentence is **'i like visiting New York City when i was 13 years old.'**, now i mask the 'New York City',which is like this:

**'i like visiting \<mask> \<mask> \<mask> when i was 13 years old.'**

Now i want to do contextualized infilling to predict these words.

First, I use Roberta to predict，this is my code:

```
from transformers import RobertaForMaskedLM, RobertaTokenizer
tokenizer = RobertaTokenizer.from_pretrained('distilroberta-base')
model = RobertaForMaskedLM.from_pretrained('distilroberta-base')

sentence = 'i like visiting New York City when i was 13 years old.'
sentence_list = sentence.split()
for i in range(3,6):
    sentence_list[i] = tokenizer.mask_token
sentence = ' '.join(sentence_list)
inputs = tokenizer(sentence, return_tensors='pt')
outputs = model(**inputs)[0]

masked_idx = (inputs['input_ids'][0] == tokenizer.mask_token_id).nonzero()
masked_idx = masked_idx.squeeze().tolist()

logits = outputs[0,masked_idx[0]:masked_idx[-1]+1,:]
probs = logits.softmax(dim=-1)
values, predictions = probs.topk(50, dim=-1)
```

And then i check the most probable words predicted by Roberta: **my amaznam museum**, so the predicted sentence is **'i like visiting ***my amaznam museum*** when i was 13 years old.'**

Maybe it make little sense, but i want to predict a sentence with more frequency here.

The self-supervised training task in the SpanBert is like this, but their model in the huggingface's transformers lib didn't train the AutoModelForMaskedLM part.

 
thanks for help
